site/ic_data_repo/tasks.py

```diff
@@ -93,7 +93,35 @@ def export_record_to_symplectic(record) -> None:
     for doi, type_id in related_work_doi.items():
         related_object_ids = client.fetch_related_objects(doi)
         for related_object_id in related_object_ids:
-            client.link_related_records(object_id, related_object_id, type_id)
+            client.link_related_records(
+                object_id, related_object_id, type_id, to_object_type="publication"
+            )
+
+    funding = metadata.get("funding", [])
+
+    for fund in funding:
+        award = fund.get("award", {})
+        award_id = None
+        award_id_type = None
+
+        if award.get("id"):
+            award_id = award["id"]
+            award_id_type = "institution-reference"
+        elif award.get("number"):
+            award_id = award["number"]
+            award_id_type = "funder-reference"
+
+        if award_id and award_id_type:
```

Comment:
I believe this works and is a bit tighter.

```suggestion
        if award_id := award.get("id"):
            award_id_type = "institution-reference"
        elif award_id := award.get("number"):
            award_id_type = "funder-reference"
        
        if award_id:
```